packages/contracts/test/factory/dao-factory.ts

```diff
@@ -187,305 +211,142 @@ describe('DAOFactory: ', function () {
-      tx = tx.to
-        .emit(createdDao, EVENTS.Granted)
-        .withArgs(
-          item,
-          daoFactory.address,
-          createdDao.address,
-          createdDao.address,
-          PermissionManagerAllowFlagAddress
-        );
-    });
-
-    tx = tx.to
-      .emit(createdDao, EVENTS.Granted)
+    await expect(tx)
+      .to.emit(daoContract, EVENTS.Revoked)
       .withArgs(
-        SET_CONFIGURATION_PERMISSION_ID,
+        ROOT_PERMISSION_ID,
         daoFactory.address,
-        createdDao.address,
-        voting.address,
-        PermissionManagerAllowFlagAddress
-      )
-      .to.emit(createdDao, EVENTS.Revoked)
-      .withArgs(
-        DAOPermissions[1],
         daoFactory.address,
-        daoFactory.address,
-        createdDao.address
-      )
-      .to.emit(createdDao, EVENTS.Granted)
-      .withArgs(
-        EXECUTE_PERMISSION_ID,
-        daoFactory.address,
-        voting.address,
-        createdDao.address,
-        PermissionManagerAllowFlagAddress
+        dao
       );
-
-    // ===== Test if user can create a vote and execute it ======
-
-    // should be only callable by ERC20Voting
-    await expect(createdDao.execute(0, [])).to.be.revertedWith(
-      customError(
-        'Unauthorized',
-        createdDao.address,
-        createdDao.address,
-        ownerAddress,
-        EXECUTE_PERMISSION_ID
-      )
-    );
-
-    await expect(voting.setConfiguration(1, 2, 3)).to.be.revertedWith(
-      customError(
-        'DaoUnauthorized',
-        createdDao.address,
-        voting.address,
-        voting.address,
-        ownerAddress,
-        SET_CONFIGURATION_PERMISSION_ID
-      )
-    );
-
-    const actions = [
-      {
-        to: actionExecuteContract.address,
-        value: 0,
-        data: actionExecuteContract.interface.encodeFunctionData('setTest', []),
-      },
-      {
-        to: voting.address,
-        value: 0,
-        data: voting.interface.encodeFunctionData(
-          'setConfiguration',
-          [3, 4, 5]
-        ),
-      },
-    ];
-
-    await voting.createVote('0x', actions, 0, 0, false, VoteOption.Yes);
-
-    expect(await voting.vote(0, VoteOption.Yes, true))
-      .to.emit(createdDao, EVENTS.Executed)
-      .withArgs(voting.address, 0, [], [])
-      .to.emit(voting, EVENTS.ConfigUpdated)
-      .withArgs(3, 4, 5);
-
-    expect(await actionExecuteContract.test()).to.equal(true);
   });
 
-  it('creates AllowlistVoting DAO', async () => {
-    let tx = await daoFactory.createAllowlistVotingDAO(
-      {
-        name: daoDummyName,
-        metadata: daoDummyMetadata,
-      },
-      dummyVoteSettings,
-      [ownerAddress],
-      zeroAddress
-    );
-
-    const {
-      dao: createdDao,
-      voting,
-      creator,
-      name,
-    } = await getDeployments(tx, false);
-
-    expect(name).to.equal(daoDummyName);
-
-    expect(creator).to.equal(ownerAddress);
+  it('makes sure all DAO permissions are correctly granted', async () => {
```

Comment:
```suggestion
  it('emits an event containing the correct permissions to be set in the created DAO', async () => {
```
It might be better to not only check the emitted event but to check if they have really been applied in the created DAO by querying `PermissionManager`.

Reply:
Generally "makes sure" should be replaced by a more fitting verb describing the behavior of the functionality being tested explicitly. Also, use active voice instead of passive voice.
Also later:
`makes sure all temporarly granted permissions are revoked` -> `revokes all temporarly granted permissions`

Reply:
_It might be better to not only check the emitted event but to check if they have really been applied in the created DAO by querying PermissionManager_

i added that to `correctly creates a DAO with one plugin` should we add it to all tests?

Reply:
You can decide for yourself what makes the most sense.